### PR TITLE
Edit Enterprise 1.9 release notes

### PR DESCRIPTION
**Pull request by pierwill**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 1]
```diff
@@ -9,28 +9,26 @@ menu:
...
+-----------
 
 ### Features
 - Support user-defined *node labels*.
   Node labels allows cluster operators to assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
-- Improve diagnostics for license problems.
-- Ingress metrics now work.
+- Improve diagnostics for license problems. Add license expiration date to `debug/vars` metrics.
+- Improved ingress metrics.
```
Comment (timhallinflux) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:20]: Where is this explained in more detail?  I think folks will want a pointer to this....

**Review by timhallinflux — CHANGES_REQUESTED** [review]

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md (hunk 1 above)
Comment (pierwill) [reply content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md]: Working on adding docs for this now.

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
-
-### Bugfixes
-- Show databases now checks read write permissions. <!-- https://github.com/influxdata/plutonium/pull/3341 -->
-- Anti-entrpoy: Update `tsm1.BlockCount()` call to match signature <!-- https://github.com/influxdata/plutonium/pull/3152 -->
-- Remove extraneous nil check from points writer. <!-- https://github.com/influxdata/plutonium/pull/3141 -->
-- Restore: Ensure a newline is printed after a successful copy <!-- https://github.com/influxdata/plutonium/pull/3137 -->
-- Remove extraneous nil check <!-- #3141 -->
-- Make 'entropy show' expiry times consistent with 'show-shards' <!-- #3355: fix:  -->
+- Allow [configuring Flux query resource usage](enterprise_influxdb/v1.9/administration/config-data-nodes/#flux-controller) (concurrency, memory, etc.).
+- Upgrade to Flux v0.113.0. This includes:
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:22]: Will, typically we link to the Flux release notes....let's double-check these release notes--looks like there may be some discrepancy between what's listed here and what's actually in the release notes?

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md (hunk 1 above)
Comment (pierwill) [reply content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md]: Have added a link to new config docs for this. See also https://github.com/influxdata/docs-v2/pull/2529.

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 3]
```diff
@@ -1,36 +1,45 @@
...
     name: Release notes
     weight: 10
     parent: About the project
 ---
 
-v1.9.0 [unreleased]
--------------------
+## v1.9.0 [TK]
+
+The release of InfluxDB Enterprise 1.9 is different from previous InfluxDB Enterprise releases
```
Comment (lesam) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:14]: I don't see https://github.com/influxdata/docs-v2/issues/2169 called out here

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
--------------------
+## v1.9.0 [TK]
 
 ### Features
 - Support user-defined *node labels*.
   Node labels allows cluster operators to assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
-- Improve diagnostics for license problems.
-- Ingress metrics now work.
+- Improve diagnostics for license problems. Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
```
[the remark this answers is not in the document]
Comment (lesam) [reply content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:22]: `license-cloud` (marketplace entitlements) and `license-file` (offline license validation) are also possible

**Review by sanderson — APPROVED** [review]

**Pull request merged**